server/lib/transferwise.ts

```diff
@@ -177,3 +177,29 @@ export const getTransfer = async (token: string, transferId: number): Promise<an
     throw new Error(message);
   }
 };
+
+export const getAccountRequirements = async (token: string, quoteId: number): Promise<any> => {
+  try {
+    const response = await axios.get(`${config.transferwise.api}/v1/quotes/${quoteId}/account-requirements`, {
+      headers: { Authorization: `Bearer ${token}` },
+    });
+    return getData(response);
+  } catch (e) {
+    const message = `Unable to get account requirements data: ${getAxiosError(e)}`;
+    logger.error(message, { quoteId });
+    throw new Error(message);
```

Comment:
Can error include sensitive information? If that's the case, we should log the full error but return something like this:
> An unknown error happened with Transferwise. Please contact support@opencollective.com.

Reply:
Agreed.

Reply:
I'm not completely sure, but since this error is generated by TW there's no way to guarantee. I agree with the solution you proposed.